ecms_base/features/custom/ecms_paragraphs/config/install/core.base_field_override.paragraph.media_item.created.yml

```diff
@@ -10,7 +10,7 @@ bundle: media_item
 label: 'Authored on'
 description: 'The time that the Paragraph was created.'
 required: false
-translatable: false
+translatable: true
```

Comment:
Should the created time be translatable?

Reply:
Ah good catch, reverted.